spec/dummy/app/themes/default/views/thredded/shared/_top_nav.html.erb

```diff
@@ -2,31 +2,39 @@
   <ul>
     <!-- always -->
     <li class="user-navigation--search">
-      <!-- <svg viewBox="0 0 32 32"><use xlink:href="#search1"></use></svg> -->
```

Comment:
So I was going to bring back these icons but the ones that you have now aren't great.